e2e/single-cluster/sharding_test.go

```diff
@@ -79,27 +79,38 @@ var _ = Describe("Filtering events by shard", Label("sharding"), Ordered, func()
 
 				By("checking the gitjob pod has the same nodeSelector as the sharded controller")
 				Eventually(func() bool {
-					shardNodeSelector, err := k.Namespace("cattle-fleet-system").Get("pods", "-o=jsonpath={.items[0].spec.nodeSelector}", "-l", "app=fleet-controller", "-l", fmt.Sprintf("fleet.cattle.io/shard-id=%s", shard))
+					shardNodeSelector, err := k.Namespace("cattle-fleet-system").Get(
```

Comment:
nit: I'm fine with reading long wrapped lines. Especially if it's the equivalent of a shell command... 🤷